dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -3693,6 +3719,132 @@ public void testParameterAnnotationTakesPrecedenceOverMethodPrefix() {
         assertEquals(19L, nine.numberId);
     }
 
+    @Test
```

Comment:
Need a comment that can go into the feature test summary. Something like:
```suggestion
    /**
     * Tests entity attribute names from embeddables and MappedSuperclass that
     * can have delimiters. Includes tests for name collisions with attributes from an
     * embeddable or superinteface. This test uses unannotated entities.
     */
    @Test
```